test/bin/scenario.sh

```diff
@@ -211,6 +212,25 @@ remove_vm() {
     rm -rf "${SCENARIO_INFO_DIR}/${SCENARIO}/vms/${vmname}"
 }
 
+
+# Function to report the full current version, e.g. "4.13.5"
+current_version() {
+    # OCP_VERSION := 4.14.0-0.nightly-2023-06-30-131338
+    grep OCP_VERSION "${MAKE_VAR_FILE}" | awk '{print $3}' | cut -f1 -d-
```

Comment:
We parse the version within the Makefile, too. Maybe we should add targets there to print out the values we need so we're only doing the parsing in 1 place?

Reply:
fyi: we already have https://github.com/openshift/microshift/blob/main/Makefile#L87-L93 so we might extend that and grep specific variable?

Reply:
The image build script looks at the RPM in the local repo. I think that's going to be safer long term, since that's the actual package we're trying to install. I'll rework things so that script and scenario.sh use the same logic.